js/npm_import.bzl

```diff
@@ -116,14 +117,10 @@ def npm_import(
         deps: A dict other npm packages this one depends on where the key is the package name and value is the version
         transitive_closure: A dict all npm packages this one depends on directly or transitively where the key is the
             package name and value is a list of version(s) depended on in the closure.
-        indirect: If True, this is a indirect npm dependency which will not be linked as a top-level node_module.
-        link_package_guard: When explictly set, check that the generated link_js_package() macro from
-            link_js_package.bzl is called within the specified package.
-
-            Default value of "." implies no guard.
-
-            This is set by automatically when using translate_pnpm_lock via npm_import to guard against linking the
-            generated node_modules into the wrong location.
+        root_path: The rootpackage where the node_modules virtual store is linked to.
```

Comment:
nit: rootpackage one word?

we'll have to add content here as we get confused adopters I'm sure